### ISSUE TITLE
Extracting audio stream from aaf?

### ISSUE THREAD

**Issue by ginwakeup** [markreidvfx/pyaaf2#89]
Hello,

I have an aaf with embedded media stored in it, and I am trying to use pyaaf2 to read clips and read their audio source/stream.
Any suggestion about how can I do that?

Thanks.

**Comment by markreidvfx** [comment]
SourceMobs have a convenience export to wav method
https://github.com/markreidvfx/pyaaf2/blob/master/aaf2/mobs.py#L391

Here are some more examples
https://github.com/markreidvfx/pyaaf2/blob/master/tests/test_import.py

You can also access any essenceData stream much like a python file object if needed 
https://github.com/markreidvfx/pyaaf2/blob/master/tests/test_essence.py

**Comment by ginwakeup** [comment]
Thank you very much, that's super helpful. 
I'd have 2 more questions if you don't mind.

1. The essenceData I am accessing is supposed to contain LTC timecode (we embedded it on purpose in the aaf). 
I noticed that if I dig enough in the SourceMob:
```
mobs = aFile.content.sourcemobs()
for source_mob in mobs:
    timeline_slot = source_mob.slots[0]
    segment = timeline_slot.segment
```
segment seems to be another SourceClip with few properties such as StartTime and Length, am I correct to think that this startTime and Length properties are evaluated from the mob essence data (which is, as I mentioned before, LTC timecode) ? Or is it just timestamp information about the session from which the aaf was exported?

2. If the answer to the previous question is no, then I probably have to evaluate the essence stream myself to extract timecode. But every SourceMob name seems to be equal, so I was wondering if there's a way to get their masterMob name? (Basically the name these clips had set in the software session).
3. 
Thank you.
